Gemfile.lock

```diff
@@ -407,7 +407,7 @@ DEPENDENCIES
   whenever
 
 RUBY VERSION
-   ruby 2.7.3p183
+   ruby 2.7.2p137
 
 BUNDLED WITH
-   2.2.16
+   2.1.4
```

Comment:
Unless you're adding/explicitly updating gem versions, I don't think this file should have been changed - I'm guessing it's from the ruby downgrade you included?